docs/how-to/authentication-plugin-examples.mdx

```diff
@@ -81,6 +101,10 @@ Amplication offers several authentication plugins to choose from. Each plugin ha
 - Enables a straightforward authentication scheme built into the HTTP protocol.
 - Requires sending user's credentials in the form of a username and password, encoded in base64, included in the Authorization header of the request.
 
+:::note
+If you use the Basic Auth Provider plugin, your service comes with one user with the username `admin` and the password `admin` by default.
```

Comment:
Not sure, but I think that with each auth plugin, we supply the admin/admin user.
Can you please check for all the auth plugins including the .NET one?